lib/chef/resource/gem_package.rb

```diff
@@ -53,7 +86,7 @@ class GemPackage < Chef::Resource::Package
         default: lazy { Chef::Config[:clear_gem_sources] }, desired_state: false
 
       property :gem_binary, String, desired_state: false,
-        description: "The path of a gem binary to use for the installation. By default, the same version of Ruby that is used by #{Chef::Dist::PRODUCT} will be installed."
+        description: "The path of a gem binary to use for the installation. By default, the same version of Ruby that is used by #{Chef::Dist::PRODUCT} will be used."
```

Comment:
> By default, the same version of Ruby that is used by #{Chef::Dist::PRODUCT} will be installed.

I updated this sentence because it made it sound like ruby was getting installed.